src/util/contentful/richTextRenderer.js

```diff
@@ -58,31 +58,11 @@ export function richTextRenderer(content) {
 		const isButton = contentfulEntryNode?.data?.target?.sys?.contentType?.sys?.id === 'button';
 		if (isRichTextContent) {
 			const richTextHTML = richTextRenderer(contentfulEntryNode?.data?.target?.fields?.richText);
-			return `
-				<div class="tw-prose tw-whitespace-pre-wrap">${richTextHTML}</div>
-			`;
+			return `<div>${richTextHTML}</div>`;
 		}
 		if (isButton) {
-			const analyticsClickEvent = contentfulEntryNode?.data?.target?.fields?.analyticsClickEvent;
-			const webClickEventName = contentfulEntryNode?.data?.target?.fields?.webClickEventName;
-
-			const analyticsDirective = () => {
-				return analyticsClickEvent ? `v-kv-track-event="['${analyticsClickEvent.join("','")}']"` : '';
-			};
-
-			const clickFunctionality = () => {
-				if (webClickEventName) {
-					return `@click="buttonClick('${webClickEventName}', $event)"`;
-				}
-				return `href="${contentfulEntryNode?.data?.target?.fields?.webLink ?? '#'}"`;
-			};
-			return `
-				<kv-button
-					variant="${contentfulEntryNode?.data?.target?.fields?.style ?? 'primary'}"
-					${analyticsDirective()}
-					${clickFunctionality()}
-				>${contentfulEntryNode?.data?.target?.fields?.label ?? ''}</kv-button>
-			`;
+			const buttonObject = JSON.stringify(contentfulEntryNode?.data?.target?.fields).replace(/"/g, '\'');
+			return `<button-wrapper class="tw-whitespace-normal" :content="${buttonObject}"/>`;
```

Comment:
Purge doesn't recognize this class, so I manually added it to the safelist. We want `tw-whitespace-pre-wrap` to support soft breaks from contentful but this breaks the buttons, so we override on the buttons with `tw-whitespace-normal`